src/containers/FourFieldForm.js

```diff
@@ -0,0 +1,47 @@
+import React, { useState } from 'react';
+
+// Components
+import SubmitButton from '../components/submitButton';
+import InputField from '../components/input-field';
+
+// helpers
+// TODO: not sure how to fix this.
+import submitForm from '../helpers/index';
```

Comment:
Not sure how to fix this.

Reply:
If you are calling the index file you don't need to identify it. Just leave the last part off. React knows to find it. When calling [directory]/[file] if the file is left off it will always default to index file in the dir. 

'../helpers/index' -> '../helpers'

This does come with complications, typically created by a dev. 

In the following setup:
src/helpers <- dir
src/helpers.js(x) <- file

Unlike what was previously mentioned; if you call 'src/helpers' with this architecture, React will pick up the file and ignore the directory. This is an example of always being sure of your architecture and ensure there are no clashes.

Reply:
There was an eslint error, it was https://github.com/benmosher/eslint-plugin-import/blob/v2.22.1/docs/rules/no-named-as-default.md and i was not able to fix it.